rust/tool/src/install.rs

```diff
@@ -161,7 +162,8 @@ impl Installer {
         // Sync files to persistent storage. This may improve the
         // chance of a consistent boot directory in case the system
         // crashes.
-        sync();
+        let boot = File::open(&self.esp_paths.esp).context("Failed to open ESP root directory.")?;
+        syncfs(boot.as_raw_fd()).context("Failed to sync ESP filesystem.")?;
```

Comment:
I was going to format these over two lines to match the other `.context(...)?` lines, but rustfmt disagrees.